mindsdb/api/http/namespaces/agents.py

```diff
@@ -275,12 +279,6 @@ def post(self, project_name, agent_name):
         if 'mode' not in existing_agent.params and any(skill.type == 'retrieval' for skill in existing_agent.skills):
             existing_agent.params['mode'] = 'retrieval'
 
-        # get model details
-        session = SessionController()
-        model_name_no_version, version = db.Predictor.get_name_and_version(existing_agent.model_name)
-        agent_model = session.model_controller.get_model(model_name_no_version, version=version, project_name=project_name)  # noqa
-        model_using = agent_model.get('problem_definition', {}).get('using', {})
-
```

Comment:
These changes were introduced as part of https://github.com/mindsdb/mindsdb/pull/9496/files but they are not compatible with non-langchain model agents